draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -176,27 +176,50 @@ conceptual level, abstracting from the data formats described afterwards.
 
 ## Creating an SD-JWT
 
-An SD-JWT, at its core, is a digitally signed document containing hash digests over the claim values with unique random salts and other metadata.
+An SD-JWT, at its core, is a digitally signed document containing digests over the claim values with random salts and other metadata. 
 It MUST be digitally signed using the issuer's private key.
 
 ```
 SD-JWT-DOC = (METADATA, SD-CLAIMS)
 SD-JWT = SD-JWT-DOC | SIG(SD-JWT-DOC, ISSUER-PRIV-KEY)
 ```
 
-`SD-CLAIMS` can be a simple object with claim names mapped to hash digests over the claim values with unique random salts:
+`SD-CLAIMS` is an object with claim names (`CLAIM-NAME`) mapped to the digests over the claim values (`CLAIM-VALUE`) with random salts (`SALT`). Digests are calculated using a digest derivation function such as a hash function, HMAC, or other (`DIGEST-DERIVATION()`):
+
 ```
 SD-CLAIMS = (
-    CLAIM-NAME: HASH(SALT | CLAIM-VALUE)
-)*
+    CLAIM-NAME: DIGEST-DERIVATION(SALT, CLAIM-VALUE)
+)
 ```
 
-The claim name (`CLAIM-NAME`) is an optional
+When an HMAC or another type of digest derivation function is used for digest derivation, a secret cryptographic key or other cryptographic secret is used instead of a salt value. 
```

Comment:
```suggestion
When an HMAC or another type of derivation function is used for digest calculation, a secret cryptographic key or other cryptographic secret is used instead of a salt value. 
```